README.md

```diff
@@ -0,0 +1,32 @@
+# JDT Debug
+
+JDT Debug implements Java debugging support and works with any JDPA-compliant target Java VM.
+It is implemented on top of the language independent "debug model" provided by the platform debugger. 
+JDT debug provides the following debugging features:
+
+- Launching of a Java VM in either run or debug mode
+- Attaching to a running Java VM
+- Expression evaluation in the context of a stack frame
+- Scrapbook pages for interactive Java code snippet evaluation
+- Dynamic class reloading where supported by Java virtual machine
+
+For more information about JDT Debug refer to the [JDT Debug website](https://eclipse.dev/eclipse/debug/index.php).
+For general information about JDT and important links, refer to the [JDT wiki page](https://github.com/eclipse-jdt/eclipse.jdt.core/wiki) or the [JDT project overview page](https://projects.eclipse.org/projects/eclipse.jdt).
+
+## Contributing
```

Comment:
does this file need a Contributing section? The existing of Contributing.md should be self explaining

Reply:
I agree with you on this. looks good without Contributing section 
<img width="730" alt="image" src="https://github.com/user-attachments/assets/09bcb852-1c0b-42aa-9797-52ae852e1252">

Reply:
> does this file need a Contributing section? The existing of Contributing.md should be self explaining

The `contributing.md` is now removed and even if it's inherited from the organization I think it's good to have a few words about contributions and to provide the link to the orga's contributing section explicitly.

Furthermore I wanted to add the Oomph setup button for the setup to be added for repository (see https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/2430) in that section in a follow-up PR, similar to how it is e.g. in the platform repository:
https://github.com/eclipse-platform/eclipse.platform.ui/?tab=readme-ov-file#how-to-contribute

Therefore I would be in favor to keep this.